framework/models.py

```diff
@@ -0,0 +1,69 @@
+# External Libraries
+from pony.orm import Set, Database, Required, PrimaryKey, db_session, Optional
+
+db = Database()
+
+
+class Base:
+    @classmethod
+    @db_session
+    def exists(cls, id_: str):
+        return cls.get(id=id_) is not None  # pylint: disable=no-member
+
+    @classmethod
+    @db_session
+    def get_s(cls, arg):
+        return cls.get(id=arg)  # pylint: disable=no-member
+
+    @property
+    @db_session
+    def json(self):
+        return self.__class__[self.id].to_dict(with_collections=True, exclude="password")  # pylint: disable=no-member
+
+
+class Mod(db.Entity, Base):
+    id = PrimaryKey(str)
+    title = Required(str)
+    icon = Optional(str, nullable=True)
+    path = Required(str)
```

Comment:
What's this for? All mods are accessed via IDs on frontend.

Reply:
If you'd checked the source you can see that I internally save it under a different path so if there's ever a security issue, people can't simply delete a mod by ID.

Reply:
You'll probably need to clarify this under the documentation